modules/ROOT/examples/live-demos/footnotes/index.html

```diff
@@ -0,0 +1,27 @@
+      <li>A superscripted footnote number is added to the document.</li>
+      <li>And the insertion point is moved down to a new entry in the footnotes section<sup id="footnote_795387001171662097474091" class="mce-footnote"><a href="#footnotes_entry_795387001171662097474091">3</a></sup> at the bottom of the TinyMCE editor.</li>
+      <li>Add text to this new footnote entry<sup id="footnote_83105724181662097486697" class="mce-footnote"><a href="#footnotes_entry_83105724181662097486697">4</a></sup>.</li>
+    </ol>
+    <p>Each superscripted footnote number links to its respective footnote entry, and each footnote entry links back to its respective superscripted footnote number. This allows for navigation between the editor content and the footnotes section.</p>
+    <p>Adding a footnote in a editor automatically updates each footnote number in the text and in the footnotes section.</p>
+    <p>For example, adding a new footnote between already extant footnotes <sup><strong>1</strong></sup> and <sup><strong>2</strong></sup> results in the new footnote becoming footnote <sup><strong>2</strong></sup> and the previous footnote <sup><strong>2</strong></sup> automatically becoming <sup><strong>3</strong></sup>.
+    <p>Superscripted footnote numbers<sup id="footnote_6878577521662090521962" class="mce-footnote"><a href="#footnotes_entry_6878577521662090521962">5</a></sup> can also be dragged to a different position in the document. Footnote numbers and their relationship with the respective footnote entry will be automatically updated.</p>
+    <p>Footnotes are available for TinyMCE as a Premium Plugin. To test Footnotes in your own TinyMCE environment, sign up for a Tiny account. New Tiny accounts include 14-day free trials of all Tiny Premium Plugins, including Footnotes.</p>
+    <div class="mce-footnotes"><hr>
+      <ol>
+        <li id="footnotes_entry_34327763141662096813220"><a class="mce-footnotes-backlink" href="#footnote_34327763141662096813220">^&nbsp;</a><span class="mce-footnotes-note">The Footnotes plugin works with TinyMCE 6.2 or later and is available with all TinyMCE Premium Plans.</span></li>
+        <li id="footnotes_entry_46102500311662090427490"><a class="mce-footnotes-backlink" href="#footnote_46102500311662090427490">^&nbsp;</a><span class="mce-footnotes-note">If there is a highlighted selection when a footnote is added, the new footnote is added to the end of the highlighted selection.&nbsp;</span></li>
+        <li id="footnotes_entry_795387001171662097474091"><a class="mce-footnotes-backlink" href="#footnote_795387001171662097474091">^&nbsp;</a><span class="mce-footnotes-note">This is a footnote entry. If the TinyMCE document is large enough to produce a scroll bar, clicking the caret prefixing this entry will scroll the document back to the line containing entry’s superscripted footnote number.</span></li>
```

Comment:
Should we mention that it won't move the cursor, though?

Reply:
Added *NOTE* admonition documenting this to `footnotes.adoc`.